src/app/public/modules/action-hub/action-hub.component.html

```diff
@@ -0,0 +1,34 @@
+<sky-wait
+  [isFullPage]="true"
+  [isWaiting]="loading"
+></sky-wait>
+<sky-fluid-grid *ngIf="!loading">
+  <sky-row>
+    <sky-column>
+      <h1 class="sky-font-heading-1">{{ title }}</h1>
```

Comment:
I know it can be a pain - but it has been discussed team wide before - and the `class` portion of this should be on its own line in the same way that we do for tags with multiple attributes. This is true everywhere except for structural directives such as `*ngIf` and `*skyThemeIf`.